pkg/operator/staticpod/controller/monitoring/manifests/service-monitor.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: monitor
+  namespace: {{ .TargetNamespace }}
```

Comment:
Because we only have unstructured for now, I'm ok using a golang template.  With real objects, I prefer real assignment in golang code.